### PR TITLE
Better readme

### PR DESCRIPTION
**Pull request by LouisJULIEN**
I found it super confusing to find out how to use the calendar experiment based on the Readme (calendar/README.md) most links redirect to. I added a few words to help.

### CODE REVIEW

**Comment by kewisch** [comment]
Superb, thanks. Let me know how things go with the calendar API, I'd love to get this into core eventually.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 **/web-ext-artifacts
 **/*.xpi
 node_modules
+.idea/
```
Comment (kewisch) [inline .gitignore:4]: Ah can you remove this? If this is an issue it should probably rather go into your ~/.gitignore_global

**Review by kewisch — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (LouisJULIEN) [reply .gitignore:4]: Not really an issue, I'm reverting that!

Comment (LouisJULIEN) [reply .gitignore]: And thanks for the gitignore_global tip

**Review by kewisch — APPROVED** [review]

**Comment by kewisch** [comment]
Needs a review/rubber stamp from @jobisoft as well.

**Pull request merged**